tests/configs/modules/compliments/compliments_cron_entry.js

```diff
@@ -0,0 +1,19 @@
+let config = {
+	modules: [
+		{
+			module: "compliments",
+			position: "middle_center",
+			config: {
+				updateInterval: 1000 * 5, // Update every 5 secs
```

Comment:
did not see this one at first review, can you please remove this updateInterval too ...